### PR TITLE
Update docs and config files - 1.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: docs/en/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+The package could be installed with composer:
+
+```
+composer require yiisoft/log-target-db --prefer-dist
+```
+
+## Migration
+
+The package provides a migration that creates the cache table for default `{{%log}}`. You can use it as follows:
```
Comment (darkdef) [inline docs/en/README.md:18]: Need add files with raw sql for direct create tables and add information about it in this readme

File: docs/en/migration/schema-mysql.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/**
+ * Database schema required by yiisoft/log-target-db for MySQL.
+ */
+DROP TABLE IF EXISTS `log`;
```
Comment (darkdef) [inline docs/en/migration/schema-mysql.sql:4]: Drop - it's bad practice. If table exists - all data will be lost

**Review by darkdef — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -8,100 +8,36 @@
...
-[![Code Coverage](https://scrutinizer-ci.com/g/yiisoft/log-target-db/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/yiisoft/log-target-db/?branch=master)
+[![codecov](https://codecov.io/gh/yiisoft/log-target-db/branch/master/graph/badge.svg?token=AP7VK8ZYIF)](https://codecov.io/gh/yiisoft/log-target-db)
 [![Mutation testing badge](https://img.shields.io/endpoint?style=flat&url=https%3A%2F%2Fbadge-api.stryker-mutator.io%2Fgithub.com%2Fyiisoft%2Flog-target-db%2Fmaster)](https://dashboard.stryker-mutator.io/reports/github.com/yiisoft/log-target-db/master)
 [![static analysis](https://github.com/yiisoft/log-target-db/workflows/static%20analysis/badge.svg)](https://github.com/yiisoft/log-target-db/actions?query=workflow%3A%22static+analysis%22)
 [![type-coverage](https://shepherd.dev/github/yiisoft/log-target-db/coverage.svg)](https://shepherd.dev/github/yiisoft/log-target-db)
 
 This package provides the Database target for the [yiisoft/log](https://github.com/yiisoft/log) library.
 
-## Requirements
+## Support databases:
```
Comment (samdark) [inline README.md:18]:
```suggestion
## Supported databases
```

File: docs/en/README.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
+# Getting started
```
Comment (samdark) [inline docs/en/README.md:1]: I don't like that the basics are moved into `docs`.

File: docs/en/migration/schema-mssql.sql [hunk 5]
```diff
@@ -0,0 +1,17 @@
+/**
+ * Database schema required by yiisoft/log-target-db for MSSQL.
+ */
+CREATE TABLE [dbo].[log] (
```
Comment (samdark) [inline docs/en/migration/schema-mssql.sql:4]: Is it needed considering we have a `DbHelper`?

**Review by samdark — APPROVED** [review]

**Pull request merged**